bundles/org.openhab.binding.siemensrds/src/test/java/org/openhab/binding/siemensrds/test/RdsTestData.java

```diff
@@ -0,0 +1,3416 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.siemensrds.test;
+
+import static org.junit.Assert.*;
+import static org.openhab.binding.siemensrds.internal.RdsBindingConstants.*;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.QuantityType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.library.unit.ImperialUnits;
+import org.eclipse.smarthome.core.library.unit.SIUnits;
+import org.eclipse.smarthome.core.types.State;
+import org.junit.Test;
+import org.openhab.binding.siemensrds.internal.RdsAccessToken;
+import org.openhab.binding.siemensrds.internal.RdsCloudException;
+import org.openhab.binding.siemensrds.internal.RdsDataPoints;
+import org.openhab.binding.siemensrds.internal.RdsPlants;
+import org.openhab.binding.siemensrds.internal.RdsPlants.PlantInfo;
+import org.openhab.binding.siemensrds.points.BasePoint;
+
+import tec.uom.se.unit.Units;
+
+/**
+ * test suite
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class RdsTestData {
+
+    /*
+     * note: temperature symbols with a degree sign: the MVN Spotless formatter
+     * trashes the "degree" (looks like *) symbol, so we must escape these symbols
+     * as octal \260 or unicode \u00B00 in the following JSON test strings
+     * 
+     * note: (at)formatter on/off tags instruct spotless not to reformat the JSON,
+     * (and perhaps also attempt to stop it trashing the degree symbols - see above)
+     */
+    //@formatter:off
+    private static final String DATAPOINTS_JSON_FULL_SET = 
```

Comment:
Data this large should not be stored in the code. Please put this data in a file and load it at runtime.

Reply:
@cpmeister I need some help with this; I would be happy to move the data into a file; but I need to know what path the file should be in, so as to ensure it is properly handled by GitHub, your MVN build environment, and the automated JUnit tests.

Reply:
Ok. I think I figured it out.